suite-common/fiat-services/src/fetch.ts

```diff
@@ -0,0 +1 @@
+export const fetchUrl = (url: string) => () => fetch(url);
```

Comment:
I don't like that you are returning function here, that's kind of counter intuitive. I would prefer to return fetch directly and wrap it in place of usage or rename this to reflect that it is returning function like `prepareFetchUrl` or something. (I like first variant better)

Reply:
821166982cc90e0934ecc2a3a8f19d24b422345a